docs/docs/migration-guide.mdx

```diff
@@ -594,9 +593,23 @@ policies:
   - name: TEDPolicy
     featurizer:
     - name: SingleStateFeaturizer
-
 ```
 
+### Class changes
+
+The following classes have moved from `rasa.core` to `rasa.shared.core`:
+
+| Rasa 1.0                             | Rasa 2.0                          |
```

Comment:
Other notable changes due to `rasa.shared`: 
* `rasa.core.Domain`
* `rasa.core.importers`
* `rasa.nlu.training_data`